### PR TITLE
Apps page in Control Center

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/control-center/apps.md [hunk 1]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
 
-The **Apps** page shows you all the apps belonging to your company and allows you to manage default app roles in your company.
+The **Apps** page shows you all the apps belonging to your company. It also allows you to manage default app roles in your company.
 
-## 2 Active and Deactivated Apps {#active-deactivated}
+The **Apps** page has the following tabs available: **Mendix Cloud**, **Mendix Free Cloud**, **Other Apps**, **Deactivated Apps**, and **Default App Roles**.
 
-### 2.1 Overview {#overview}
+{{% alert color="info" %}}
+We are working on giving each environment a separate tab. Stay tuned.
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: @Luyao-Zhang-1 
We are providing separate tabs for the Apps deployed on Mendix Cloud, Mendix Free Cloud, and Other Apps tab(For apps that are not yet deployed or for apps deployed to other cloud). So, I feel we can remove this statement.
Or
We can say 'We are working on giving a tab for apps those are deployed to private cloud'
CC : @SangamLeo

File: content/en/docs/control-center/apps.md [hunk 2]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
+
+You can export apps to an Excel file. For details, see the [Exporting Apps to an Excel File](#export-to-excel) section.
+
+## 4 Other Apps
+
+The **Other Apps** tab shows the unlicensed apps that are not deployed as well as apps that are deployed to other environments than Mendix Cloud. They are either deployed to other clouds or do not have an environment. 
 
 {{% alert color="info" %}}
-In the overview, you can activate or deactivate up to 100 apps at a time and delete 1 app at a time.
+We are working on giving each environment a separate tab. Stay tuned.
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: Same as previous comment in Introduction

File: content/en/docs/control-center/apps.md [hunk 3]
```diff
@@ -56,13 +210,33 @@ The app details page has the following tabs:
...
+| App team members can access the app code repository         | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} |
+| App stays persistent                                        | {{< figure src="/attachments/control-center/apps/check-mark.svg" >}} | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} |
+| App team members can access the app in the Developer Portal | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} |
+| Cloud environment stays persistent                          | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} | {{< figure src="/attachments/control-center/apps/cross-mark.svg" >}} |
+
+{{% alert color="info" %}}
+Deleting and deactivating apps is currently not supported for licensed apps or apps that are not running on Mendix Cloud. Therefore, you do not see these options on the **Mendix Cloud** tab or on the **Other Apps** tab if you select apps that are deployed on a different environment than Mendix Cloud.
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: @Luyao-Zhang-1 
I agree Deleting and deactivating apps is currently not supported for licensed apps 
but
`OtherApps tab consists of the apps that do not have an environment or apps deployed to other clouds`
So Deleting and deactivating apps is supported for Other Apps tab
CC: @SangamLeo

File: content/en/docs/control-center/apps.md [hunk 4]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
+
+The **Default App Roles** tab shows the default [team roles](/developerportal/general/app-roles/#team-roles) assigned for every new app created in your company. Hovering over a role brings up options to **Delete** and **Edit** the role. Click **Add Role** to add a new default role, which includes permissions on accessing [Team Server](/developerportal/general/team-server/) and [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/) information.
+
+{{< figure src="/attachments/control-center/apps/roles.jpg" >}}
+
+## 7 Filtering Apps in the Table {#filter-apps}
+
+On each tab with an table of apps, you can find filters on the top, which allows you to filter apps using different criteria. 
+
+With the filter as shown below, you can search for string matches on any of the string fields in the table, for example, the app ID, app name, or environment name. Clicking the button on the left allows you to use comparisons and string operations, such as **Contains**, **Starts with**, and **Greater than**. (This filter is available on the **Mendix Cloud**, **Mendix Free Cloud**, and **Other Apps** tabs.)
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md:185]: @Luyao-Zhang-1  
This filter is also available on **Deactivated Apps** tab

File: content/en/docs/control-center/apps.md [hunk 5]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
+
+{{< figure src="/attachments/control-center/apps/filter-apps.png" >}}
+
+With the filter as shown below, you can filter apps by the date of the last commit. Clicking the button on the left allows you to use comparisons and string operations, such as **Between** and **Empty**. (This filter is available on the **Mendix Cloud**, **Mendix Free Cloud**, and **Other Apps** tabs.)
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: This filter is also available on **Deactivated Apps** tab

File: content/en/docs/control-center/apps.md [hunk 6]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
+
+- **Environment**: the name of the environment.
+
+- **Deployed Status**: whether the app has been deployed on this environment
+
+- **Mainline Version**: the Mendix version of the app on the mainline in the repository
+
+- **Deployed Version**: the Mendix version of the deployed app
+
+- **Last Commit**: the last commit in any branch of the repository
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: @Luyao-Zhang-1  
Last Commit is the Latest Commit of MainLine

File: content/en/docs/control-center/apps.md [hunk 7]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
+
+- **EnvID**: the unique identifier of the environment.
+
+- **Deployed Status**: whether the app has been deployed on this environment
+
+- **Mainline Version**: the Mendix version of the app on the mainline in the repository
+
+- **Deployed Version**: the Mendix version of the deployed app
+
+- **Last Commit**: the last commit in any branch of the repository
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: @Luyao-Zhang-1
Last Commit is the Latest Commit of MainLine

File: content/en/docs/control-center/apps.md [hunk 8]
```diff
@@ -14,37 +14,191 @@ A member in Control Center means a user of the Mendix platform who participates
...
+
+The description of the items that you can select are as follows:
+
+- **AppID**: the unique identifier of the app
+- **App Name**: the name of the app
+
+  {{% alert color="info" %}}
+  You can view the [app details](#app-details) by clicking the app name in the table.
+  {{% /alert %}}
+- **Members**: the number of the team members of the app
```
Comment (SharishmaPodduturi999) [inline content/en/docs/control-center/apps.md]: @Luyao-Zhang-1 
The 'Other Apps' tab also contain LastCommit column

File: content/en/docs/control-center/apps.md (hunk 3 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/control-center/apps.md]: Thanks for your feedback. The sentence was indeed a bit ambiguous. I changed it to:

Therefore, you do not see these options on the **Mendix Cloud** tab; and you also do not see these options on the **Other Apps** tab if you select apps that are deployed on a different environment than Mendix Cloud.

Is this better? @SharishmaPodduturi999

File: content/en/docs/control-center/apps.md (hunk 8 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/control-center/apps.md]: @SharishmaPodduturi999  could you please share a screenshot of the Other Apps with me?

File: content/en/docs/control-center/apps.md (hunk 7 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/control-center/apps.md]: @SharishmaPodduturi999  thanks. Leotien said it is the last commit of any branch. I will double check with her.

Comment (Luyao-Zhang-1) [reply content/en/docs/control-center/apps.md]: Indeed, it is the main line. I updated it. Thank you!

**Pull request merged**